### PR TITLE
Add documentation for website

### PR DESCRIPTION
**Pull request by kevgo**
@charlierudolph @allewun @ricmatsui @redouglas 

Documentation for how to work on the website
* local development
* deployment

### CODE REVIEW

**Comment by charlierudolph** [comment]
:ship:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ This guide will help you get started and outline some things you should know whe
 * [ShellCheck](https://github.com/koalaman/shellcheck)
   * for linting the Bash code
 
+##### For website development
+* [Node.js](https://nodejs.org)
+  * on OS X best installed via [HomeBrew](http://brew.sh)
```
Comment (allewun) [inline CONTRIBUTING.md]: uncamelcase Homebrew

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -253,3 +259,18 @@ Every Git Town command
 * has a [Markdown page](./documentation/commands) that is identical to the man page
 * is listed on the [git-town man page](./man/man1/git-town.1)
 * is listed on the [README](./README.md)
+
+
+## Web Site Development
```
Comment (allewun) [inline CONTRIBUTING.md]: `s/Web Site/Website`

**Comment by allewun** [comment]
LGTM after comments. Maybe also mention what "website" is being referred to?

File: CONTRIBUTING.md (hunk 1 above)
Comment (kevgo) [inline CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (kevgo) [inline CONTRIBUTING.md]: done

**Pull request closed without merging**